.github/workflows/release-please.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
         if: ${{ steps.release.outputs.release_created }}
       - uses: actions/setup-node@v1
         with:
-          node-version: 12
+          node-version: 14
```

Comment:
shouldn't it be 16 then?

Reply:
Yes indeed... 😅
Thanks for catching this!